### ISSUE TITLE
Alarm

### ISSUE THREAD

**Issue by Arya766** [hw-manager/android#14]
There's a real need for alarms for each homework.
Alarms have to have these properties:
1. User should be able to select alarm other than the date the homework should be ready.
2. Just days are not good enough for alarms; there must be an option for hours and minutes too.
3. The numbers of alarms for each homework must be infinite.
4. There must be "snooze options too.

**Comment by AltNico** [comment]
Probably a duplicate of #3?

**Comment by Arya766** [comment]
Not exactly. #3 is about showing in the notification bar. This is about alarming a selected ringtone. You can consider these two, parts of each other.
